documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -58,214 +54,116 @@ from additional images.
-image completely replaces a directory from a previous
-image instead of combining the two directories,
-then you can customize the command that the second image uses to populate
-the shared volume in order to force a replace.
+{{% notice warning %}}
+If you set modelHome and wdtInstallHome to a non-default value,
+then the operator will ignore the WDT model and installation files from the auxiliary image(s).
+{{% /notice %}}
 
-For example, you can customize the 
-the second image's `serverPod.auxiliaryImage.command` field
-to first delete the directory that was already copied from the
-earlier image and then have it perform a normal copy:
+#### Configuration Examples
+##### Configuration Example 1: Basic Configuration
+This example specifies the required image parameter for the auxiliary image(s); all other fields are at default values.
 
-```shell
-...
+```
 spec:
-  ...
-  auxiliaryImageVolumes:
-  - name: aivolume
-    mountPath: /auxiliary
-  ...
-  serverPod:
-    auxiliaryImages:
-    - image: domain-image-A:v1
-      volume: aivolume
-    - image: domain-image-B:v1
-      volume: aivolume
-      # the following command replaces 'mydir' instead of merging it:
-      command: 'rm -fr $AUXILIARY_IMAGE_TARGET_PATH/mydir; cp -R $AUXILIARY_IMAGE_PATH/* $AUXILIARY_IMAGE_TARGET_PATH'
+  configuration:
+    model:
+      auxiliaryImages:
+      - image: model-in-image:v1
 ```
 
-#### Merge ordering example
-
-Assuming you have auxiliary images are defined at the domain, the WebLogic cluster,
-and the server scope (myserver is part of the mycluster):
-
+##### Configuration Example 2: Source Locations
+This example is same as Example 1 except that it specifies the source locations for the WebLogic Deploy Tooling model and installation files.
 ```
-...
 spec:
-  auxiliaryImageVolumes:
-  - name: aivolume
-    mountPath: /auxiliary
-  serverPod:
-    auxiliaryImages:
-    - image: domain-image-A:v1
-      volume: aivolume
-    - image: domain-image-B:v1
-      volume: aivolume
-  cluster:
-  - name: mycluster
-    serverPod:
-      auxiliaryImages:
-      - image: cl-image-A:v1
-        volume: aivolume
-      - image: cl-image-B:v1
-        volume: aivolume
-  managedServers:
-  - serverName: "myserver"
-    serverPod:
+  configuration:
+    model:
       auxiliaryImages:
-      - image: ms-image-A:v1
-        volume: aivolume
-
+      - image: model-in-image:v1
+        sourceModelHome: /foo/models
+        sourceWDTInstallHome: /bar/weblogic-deploy
 ```
 
-Then the files from the images will be merged into their shared `aivolume` volume `/auxiliary` mount path in the following order:
+##### Configuration Example 3: Multiple Images
+This example is the same as Example 1, except it configures multiple auxiliary images and sets the `sourceWDTInstallHome` 
+for the second image to `None`.
+In this case, the source location of WebLogic Deploy Tooling installation from the second image `new-model-in-image:v1` will be ignored.
```

Comment:
source location of WebLogic Deploy Tooling installation -> source location of the WebLogic Deploy Tooling installation